.github/workflows/ci.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           python-version: "3.8.x"
           architecture: "x64"
       - run: pip install -r docs/requirements-doc.txt
-      - run: cd docs && sphinx-build -T -E -b html . build
+      - run: cd docs && sphinx-build -T -E -W -b html . build
```

Comment:
With this flag docs build warnings will be transformed into errors, so warnings are tracked in CI since this moment.

Reply:
@devin-petersohn, is there an option to turn on -W for default docs build (docs/readthedocs.org:modin job)?

Reply:
I do not see a way to do it in the admin tools. Can it be done in `conf.py`?

Reply:
Given [this](https://stackoverflow.com/questions/38048945/how-to-turn-warnings-into-errors-when-building-sphinx-documentation-with-setupto#:~:text=In%20recent%20versions%20of%20Sphinx%2C%20you%20do%20this%20by%20adding%20an%20additional%20option%20to%20the%20section%20in%20setup.cfg%3A), it can be done in `setup.cfg`.

Reply:
> is there an option to turn on -W for default docs build (docs/readthedocs.org:modin job)?

Not sure we should do so, if the docs building generates some warnings we'll see it in CI anyway. We probably don't want for "production documentation" build to halt because of warnings, because if the build halts the docs are not being generated at all, so apparently  just white screen for a user trying to reach the docs instead of somehow built documentation

Reply:
The PRs which break docs build shouldn't be merged.

Reply:
+1 for @dchigarev suggestion. In my oppinion it is better to have docs built with warnings than broken at all.

> The PRs which break docs build shouldn't be merged.

Changes that introduce any docs warnings will be checked during CI (we will check it with this new `-W` flag) and won't be merged because of broken CI.

Reply:
This will work until we remove `build-docs` job. At some point we may want to get rid of a duplicate job for docs build. Imagine if it is the job named `build-docs`, but not `docs/readthedocs.org:modin`. Then, if the latter doesn't have `-W` flag turned on, we may merge a PR which introduces some warnings. Eventually, we may get a lot of warnings to be fixed as we have had by now. So, we should probably keep both the jobs for docs build.

Reply:
Ok, i see, i think it is fine to have additional check.
We can make it by setting additional flag `fail_on_warning` in the `.readthedocs.yaml` config file.